images/runtime/php-fpm/template.base.Dockerfile

```diff
@@ -10,7 +10,15 @@ ENV NGINX_RUN_USER www-data
 ENV NGINX_DOCUMENT_ROOT /home/site/wwwroot
 # Install NGINX 
 RUN apt-get update \
-    && apt-get install nano nginx -y
+    && apt-get install nano curl gnupg2 ca-certificates lsb-release debian-archive-keyring -y \
+    && curl https://nginx.org/keys/nginx_signing.key | gpg --dearmor \
+        | tee /usr/share/keyrings/nginx-archive-keyring.gpg >/dev/null \
+    && gpg --dry-run --quiet --import --import-options import-show /usr/share/keyrings/nginx-archive-keyring.gpg \
```

Comment:
I might be missing something, but if this is a dry run, then could you please clarify what is this command doing?

Reply:
It basically verifies the downloaded files contains the proper key. The step will simply output a key for us to verify and do nothing else.

Reply:
OK, thanks for clarifying